Source/Core/Core/Core.cpp

```diff
@@ -864,7 +865,8 @@ void UpdateTitle()
     }
   }
 
-  std::string message = StringFromFormat("%s | %s", SSettings.c_str(), SFPS.c_str());
+  std::string message = StringFromFormat("%s | %s | %s", Common::scm_rev_str.c_str(),
```

Comment:
Would it be too much to switch it over to libfmt while changing the string?

Reply:
There are plenty other places in this file that need the `fmt` treatment, so I'd rather leave them as-is and address them all in one fell swoop.

Reply:
That's fair.